common/src/test/resources/mappings/test-mapping-malformed.json

```diff
@@ -0,0 +1,13 @@
+{
+  "_meta": {
+    "schema_version": "1"
```

Comment:
Im noticing that this schema version is a string but the others are integers is that allowed?